tests/test_attributes.py

```diff
@@ -0,0 +1,364 @@
+from django.template import Context, Template, TemplateSyntaxError
+from django.utils.safestring import SafeString, mark_safe
+
+from django_components import component, types
+from django_components.attributes import append_attributes, attributes_to_string
+
+# isort: off
+from .django_test_setup import *  # NOQA
+from .testutils import BaseTestCase
+
+# isort: on
+
+
+class AttributesToStringTest(BaseTestCase):
+    def test_simple_attribute(self):
+        self.assertEqual(
+            attributes_to_string({"foo": "bar"}),
+            'foo="bar"',
+        )
+
+    def test_multiple_attributes(self):
+        self.assertEqual(
+            attributes_to_string({"class": "foo", "style": "color: red;"}),
+            'class="foo" style="color: red;"',
+        )
+
+    def test_escapes_special_characters(self):
+        self.assertEqual(
+            attributes_to_string({"x-on:click": "bar", "@click": "'baz'"}),
+            'x-on:click="bar" @click="&#x27;baz&#x27;"',
+        )
+
+    def test_does_not_escape_special_characters_if_safe_string(self):
+        self.assertEqual(
+            attributes_to_string({"foo": mark_safe("'bar'")}),
+            "foo=\"'bar'\"",
+        )
+
+    def test_result_is_safe_string(self):
+        result = attributes_to_string({"foo": mark_safe("'bar'")})
+        self.assertTrue(isinstance(result, SafeString))
+
+    def test_attribute_with_no_value(self):
+        self.assertEqual(
+            attributes_to_string({"required": None}),
+            "",
+        )
+
+    def test_attribute_with_false_value(self):
+        self.assertEqual(
+            attributes_to_string({"required": False}),
+            "",
+        )
+
+    def test_attribute_with_true_value(self):
+        self.assertEqual(
+            attributes_to_string({"required": True}),
+            "required",
+        )
```

Comment:
Do you mean the `attributes_to_string` function? Didn't think of that, but it does make sense to document it, so people can use the feature also outside of templates.

Reply:
@JuroOravec No, that attributes with True gets translated to a "bare" attribute.

Reply:
Updated in https://github.com/EmilStenstrom/django-components/pull/491/commits/4cbafd55576408dc9ccd258116a076cf87358a45.

I also documented the `attributes_to_string`, because I can imagine that otherwise there might be someone who would do the following just to get that behavior:
```py
Template("{% html_attrs attrs ... %}").render(Context({ "attrs": attrs, ... }))
```